### ISSUE TITLE
MST broken in current game version

### ISSUE THREAD

**Issue by Dalenacio** [chaosvolt/MST_Extra_Mod#16]

**Comment by Dalenacio** [comment]
When creating a world with MST, and then trying to create a character within that world, error messages pop up and the game refuses to launch, booting the player back to the main menu.

To be clear, this is with the inbuilt version of MST that requires no additional installation.

Link to the full stack backtrace: [debug extract.txt](https://github.com/chaosvolt/MST_Extra_Mod/files/5126607/debug.extract.txt)
----------------------
Error messages are as follows

 DEBUG    : Warning: pebble_clay loads damage using legacy methods - damage type may be wrong

 FUNCTION : bool assign(const JsonObject&, const string&, damage_instance&, bool, const damage_instance&, const damage_instance&)
 FILE     : src/assign.h
 LINE     : 861

-----------------------
DEBUG    : Warning: bearing_lead loads damage using legacy methods - damage type may be wrong

 FUNCTION : bool assign(const JsonObject&, const string&, damage_instance&, bool, const damage_instance&, const damage_instance&)
 FILE     : src/assign.h
 LINE     : 861
------------------------
 DEBUG    : Error: data/mods//More_Survival_Tools/ammo.json: line 72:22: Integers cannot have a decimal point or negative order of magnitude.

 grill and no 'real' camping trip could do its cooking without it, until the Cataclysm.  Nowadays, it is even more valuable then before, being the key component in the preparation of several military-grade explosives.",
    "weight": 16.625,
                     ^
    "volume": 2,
    "price": 1800,
    "bashing": 1,


 FUNCTION : bool main_menu::new_character_tab()
 FILE     : src/main_menu.cpp
 LINE     : 948

**Comment by chaosvolt** [comment]
You're using More Survival Tools, the old in-repo mod that was obsoleted and is out of date (as I was its only maintainer, and no longer contribute to CDDA). This is for MST Extra, its intended replacement and is a third-party mod. The features reported in those error messages (conversion of how ammo damage is specified, and conversion of item weights from potentially-incorrectly-defined integers to strings) have been implemented in this mod for several months already. I assume you were either told to remove the section in modinfo marking the old mod as obsolete (I don't recall if they've deleted it yet or left it obsolete), or obtained the old mod somewhere else.

Download the correct version of this mod for current experiential builds here, via selecting the "code" button and hitting "download zip": https://github.com/chaosvolt/MST_Extra_Mod
Or if you're using the build of 0.E Experimental immediately preceding the Nested Containers feature, a release is here: https://github.com/chaosvolt/MST_Extra_Mod/releases
And finally, if you're using 0.E Stable, I set aside a branch for it but didn't make a release yet, follow same instructions as you do when grabbing the latest master for recent experiementals: https://github.com/chaosvolt/MST_Extra_Mod/tree/0E-release

Do tell me if anything else comes up after following the requisite steps to obtain the correct mod, however.
